### DISCUSSION TITLE
Ability to use "include" for docker compose file

### DISCUSSION THREAD

**Discussion by slamer59** [coollabsio/coolify#4295] [Feature Requests]
Hello,
some deployments have different docker-compose for dev, prod, etc. 

To deploy, we can add some configuration which overide dev configuration and apply something like this: 

`docker-compose -f docker-compose.dev.yml -f docker-compose.prod.yml  up`

Pointing to **docker-compose.prod.yml**  in coolify will fail as he as no clue about docker-compose.dev.yml configuration.

Solutions : 
- custom docker compose command is possible
- add [extend to all ](https://docs.docker.com/compose/how-tos/multiple-compose-files/extends/) This solution works for me but we need to change every configuration
- add [include](https://docs.docker.com/compose/how-tos/multiple-compose-files/include/) at the top

In the last solution, include is removed ?
![image](https://github.com/user-attachments/assets/e12b92fd-7c4b-46ba-94fe-abb85b843421)

Here is one exemple : 
https://gitlab.com/noe-app/noe

Regards

**Comment by djsisson** [comment]
@slamer59 you can use that exact command in the custom start option on the project page you would need -d on teh end
